### PR TITLE
Move the existing sentence up

### PR DESCRIPTION
**Pull request by VineetSadawari**
**Summary**
We usually tend to copy the command and execute it. So, if we move the existing sentence Right after the first command, Beginners would easily understand the command while using it in Windows or Linux

Signed-off-by: Vineet Sadawari <myvineetsadawari@gmail.com>

### CODE REVIEW

File: site/content/docs/docker/about.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ You can run the Automation Framework in docker using the zap.yaml file in the cu
 ```bash
 docker run -v $(pwd):/zap/wrk/:rw -t owasp/zap2docker-stable zap.sh -cmd -autorun /zap/wrk/zap.yaml
 ```
+NOTE: Linux users can use $(pwd).Windows users can replace ($pwd) with 'C:\your\working\directory'.
```
Comment (thc202) [inline site/content/docs/docker/about.md:80]: This is already mentioned some lines down:
```
Note that `$(pwd)` is only supported on Linux and MacOS - on Windows you will need to replace this with the full current working directory.
```

Comment (VineetSadawari) [reply site/content/docs/docker/about.md:80]: Great! 
Can we Move the line upside  i.e right after the first command?
That would be easier for anyone to understand.

**Comment by kingthorin** [comment]
I see you’ve submitted this via the web UI and thus can’t signoff. Could you confirm agreement with https://developercertificate.org ?

**Comment by VineetSadawari** [comment]
> I’m guessing you’ve submitted this change via the GitHub web UI and thus can’t signoff. Could you confirm agreement with https://developercertificate.org ?

Yes, used GitHub web UI. 
Yes., I agree with terms under https://developercertificate.org/.

**Comment by kingthorin** [comment]
Setting DCO pass.
DEBBD364-12B0-4A69-91B0-02493A9F545E

**Comment by thc202** [comment]
The pull request still needs to be updated though.

**Comment by VineetSadawari** [comment]
> The pull request still needs to be updated though.

Do i need to do anything from my side?

**Comment by thc202** [comment]
Yes, move the existing sentence up.

**Comment by VineetSadawari** [comment]
> Yes, move the existing sentence up. Pull request title and description should be updated accordingly (same for commit).

Can you please Check? Is it ok?

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you both!

**Pull request merged**